README.rst

```diff
@@ -1029,6 +1029,58 @@ You can also change the parameters used for iperf checks (e.g. timing, port, use
 password, rsa_publc_key etc) using the `OPENWISP_MONITORING_IPERF_CHECK_CONFIG
 <#OPENWISP_MONITORING_IPERF_CHECK_CONFIG>`_ setting.
 
+Iperf Check Parameters
+######################
+
+Currently, iperf check supports the following parameters:
+
++-----------------+----------+-------------------+
+| **Parameter**   | **Type** | **Default Value** |
++-----------------+----------+-------------------+
+| host            | list     | []                |
++-----------------+----------+-------------------+
+| username        | str      | ''                |
++-----------------+----------+-------------------+
+| password        | str      | ''                |
++-----------------+----------+-------------------+
+| rsa_public_key  | str      | ''                |
++-----------------+----------+-------------------+
+| client_options  | dict     | {}                |
++-----------------+----------+-------------------+
+| port            | int      | 5201              |
++-----------------+----------+-------------------+
+| time            | int      | 10                |
++-----------------+----------+-------------------+
```

Comment:
⚠️ Before spending time on this, let's double check with @nemesisdesign. 

I would restructure this table to show that other parameters should be defined under `client_options` 

```rst
+--------------------+-------------------------------------+
| ``client_options`` | +----------+---------+------------+ |
|                    | | ``port`` | ``int`` | ``5201``   | |
|                    | +----------+---------+------------+ |
|                    | | ``time`` | ``int`` | ``10``     | |
|                    | +----------+---------+------------+ |
+--------------------+-------------------------------------+
```

Reply:
I also think a nested table will fit better for `client_options`.

Reply:
![Screenshot from 2022-08-21 20-31-27](https://user-images.githubusercontent.com/56113566/185797430-14f13453-7de1-4607-b672-f0a4c3704a24.png)

Reply:
looks good!